tajo-algebra/src/main/java/org/apache/tajo/algebra/WindowSpec.java

```diff
@@ -128,12 +128,14 @@ public boolean equals(Object obj) {
   public static enum WindowFrameStartBoundType {
```

Comment:
Thank you!
I'll remove 'static' for WindowFrameUnit, WindowFrameStartBoundType, and WindowFrameEndBoundType.